### PR TITLE
Create blog post page for "Developer Guide for Jakarta EE 10 and MicroProfile 6.0"

### LINKED ISSUE #1678: Please Publish Emily's blog on https://jakarta.ee/news
Here is Emily Jiang's blog, she submitted it using the newsroom but couldn't upload the images properly. She mentioned they keep getting lost. But we would like to have her blog on Jakarta's domain anyways. So kindly add the following content: 
[Developer Guide for Jakarta EE 10 and MP 6.0.docx](https://github.com/jakartaee/jakarta.ee/files/11491354/Developer.Guide.for.Jakarta.EE.10.and.MP.6.0.docx)

### PR DESCRIPTION
**Pull request by oliviergoulet5**
Resolves #1678

### CODE REVIEW

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+title: Developer Guide for Jakarta EE 10 and MicroProfile 6.0
+date: 2023-05-17
+publishDate: 2023-05-17
+type: announcements
+news/tags:
+  - Jakarta EE
+  - MicroProfile
+authors: [{ gh_handle: Emily-Jiang, name: Emily Jiang }]
+summary: >-
+  On 22nd September 2022, Jakarta EE 10 was released. Three months later,
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md:11]:
```suggestion
  On 22nd September 2022, [Jakarta EE 10](https://jakarta.ee/specifications/platform/10/) was released. Three months later,
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+  MicroProfile 6.0 was released with Jakarta EE 10 Core Profile alignment. We
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md:12]:
```suggestion
  [MicroProfile 6.0](https://microprofile.io/2023/01/10/microprofile-6-0-release/) was released with Jakarta EE 10 Core Profile alignment. We
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 3]
```diff
@@ -0,0 +1,785 @@
...
+  are very proud to announce that Open Liberty 23.0.0.3 supports both Jakarta
+  EE 10 and MicroProfile 6.0, and it is the first runtime that support both
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md]:
```suggestion
  EE 10 and MicroProfile 6.0, and it is the **first** runtime that support both
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 4]
```diff
@@ -0,0 +1,785 @@
...
+publishDate: 2023-05-17
+type: announcements
+news/tags:
+  - Jakarta EE
+  - MicroProfile
+authors: [{ gh_handle: Emily-Jiang, name: Emily Jiang }]
+summary: >-
+  On 22nd September 2022, Jakarta EE 10 was released. Three months later,
+  MicroProfile 6.0 was released with Jakarta EE 10 Core Profile alignment. We
+  are very proud to announce that Open Liberty 23.0.0.3 supports both Jakarta
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md:13]:
```suggestion
  are very proud to announce that [Open Liberty](https://openliberty.io/) 23.0.0.3 supports both Jakarta
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 5]
```diff
@@ -0,0 +1,785 @@
...
+  EE 10 and MicroProfile 6.0, and it is the first runtime that support both
+  Jakarta EE 10 and MicroProfile 6.0 in a production ready version. In this
+  blog, we are going to discuss some of the main features in Jakarta EE 10 and
+  MicroProfile 6.0 and then demonstrate how you can use them.
+---
+
+On 22nd September 2022, Jakarta EE 10 was released. Three months later,
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md]:
```suggestion
On 22nd September 2022, [Jakarta EE 10](https://jakarta.ee/specifications/platform/10/) was released. Three months later,
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 6]
```diff
@@ -0,0 +1,785 @@
...
+MicroProfile 6.0 was released with Jakarta EE 10 Core Profile alignment. We are
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md]:
```suggestion
  [MicroProfile 6.0](https://microprofile.io/2023/01/10/microprofile-6-0-release/) was released with Jakarta EE 10 Core Profile alignment. We
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 7]
```diff
@@ -0,0 +1,785 @@
...
+shows individual traces with multiple spans included. Each span records an
+individual operation. With the span information, if something goes wrong,
+spotting errors is no longer a challenge.
+
+{{< figure src="./images/compare-traces.jpg" >}}
+
+I have covered all MicroProfile 6.0 content. You can find the compatible
+implementation for MicroProfile 6.0 here. Open Liberty is one of the first one.
+
+### Using MicroProfile 6.0 with OpenLiberty
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md]:
```suggestion
### Using MicroProfile 6.0 with Open Liberty
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 8]
```diff
@@ -0,0 +1,785 @@
...
+        <feature>microPorfile-6.0</feature>
+</featureManager>
+```
+
+### Migration from older Jakarta EE and MicroProfile
+
+Open Liberty lists specification differences for Jakarta EE versions and
+MicroProfile versions. Refer to [this doc page](https://openliberty.io/docs/latest/reference/diff/jakarta-ee10-diff.html) 
+lists the behaviour differences from Jakarta EE 9.1 to Jakarta EE 10 and [this page](https://openliberty.io/docs/latest/reference/diff/mp-50-60-diff.html) 
+for the differences between MicroProfile 5.0 and MicroProfile 6.0.
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md]:
```suggestion
for the differences between MicroProfile 5.0 and MicroProfile 6.0. To learn more on Jakarta EE, MicroProfile as well as other cloud native technologies, checkout the [Open Liberty Guides](https://openliberty.io/guides/). To create your cloud native applications using MicroProfile 6.0 and Jakarta EE 10, use [Open Liberty starter ](https://openliberty.io/start/)to start coding.
```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 9]
```diff
@@ -0,0 +1,785 @@
...
+
+
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md:794]:
```suggestion
## Useful links

- [Jakarta EE](https://jakarta.ee/)
- [MicroProfile](https://microprofile.io/)
- [Open Liberty](https://openliberty.io/)
- Create applications using MicroProfile and Jakarta EE
  - [Open Liberty Starter](https://openliberty.io/start/)
  - [MicroProfile Starter](https://start.microprofile.io/)
  - [Jakarta EE Starter](https://start.jakarta.ee/)

```

File: content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md [hunk 10]
```diff
@@ -0,0 +1,785 @@
+---
+title: Developer Guide for Jakarta EE 10 and MicroProfile 6.0
+date: 2023-05-17
+publishDate: 2023-05-17
+type: announcements
+news/tags:
+  - Jakarta EE
+  - MicroProfile
+authors: [{ gh_handle: Emily-Jiang, name: Emily Jiang }]
```
Comment (Emily-Jiang) [inline content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md]: Can you link to my eclipse account https://projects.eclipse.org/user/8461 instead?

Comment (oliviergoulet5) [reply content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md]: Certainly. I'll update the PR some time today.

**Review by Emily-Jiang — APPROVED** [review]
LGTM! Thank you @oliviergoulet5 !

**Comment by Emily-Jiang** [comment]
@oliviergoulet5 where is the live page?

**Comment by oliviergoulet5** [comment]
> @oliviergoulet5 where is the live page?

https://jakarta.ee/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/

With this url, you can submit it into the newsroom to have this page linked.

**Comment by oliviergoulet5** [comment]
@Emily-Jiang I submitted the post via the newsroom.

**Pull request merged**